src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -83,9 +83,10 @@ class FRONTEND_API FrontEnd {
     virtual void convert(const std::shared_ptr<ov::Model>& partially_converted) const;
```

Comment:
Should we run normalize method inside "convert" (not this one, in line 79)
 /// \brief Completely convert and **normalize entire Model**, throws if it is not
/// possible

Does this description mean that if we know how to convert model, maybe not with 1:1 fw->ov mapping, but with n:1 or n:n, we should do it?

Reply:
To run or not to run `normalize` inside any other method is an FE internal detail. This doesn't need to be specified. Each method has described behavior from the user side -- only this matters. Let me just give you a couple of examples: (1) Imagine you don't have any code in `normalize`, why do you need to call it inside `convert`? (2) `convert` may be implemented in a way that runs the same normalization passed, which `normalize` does, immediately after the conversion like matcher passes that can be run in a pipeline in a single model graph pass. Technically it is possible and it doesn't require running `normalize` as a method inside `convert`.